include/util.h

```diff
@@ -6,16 +6,28 @@
 
 #pragma once
 
+#define PASTE(a, b) a ## b
+#define _STRINGIFY(a) #a
+#define STRINGIFY(a) _STRINGIFY(a)
+
+#ifdef __ASSEMBLER__
+
+/* Some assemblers don't recognise the ul/ull (unsigned long) suffix */
+#define UL_CONST(x) x
```

Comment:
ok, if this is the reason for using the macros (using the constants in assembler) it makes a lot more sense, can you add that to the initial commit?

Reply:
done